rocAL/rocAL_pybind/amd/rocAL/readers.py

```diff
@@ -0,0 +1,101 @@
+import rali_pybind as b
+from amd.rocAL.pipeline import Pipeline
+
+def coco(*inputs,file_root, annotations_file='', bytes_per_sample_hint=0, dump_meta_files=False, dump_meta_files_path='', file_list='', initial_fill=1024,  lazy_init=False, ltrb=False, masks=False, meta_files_path='', num_shards=1, pad_last_batch=False, prefetch_queue_depth=1,
+                 preserve=False, random_shuffle=False, ratio=False, read_ahead=False,
+                 save_img_ids=False, seed=-1, shard_id=0, shuffle_after_epoch=False, size_threshold=0.1,
+                 skip_cached_images=False, skip_empty=False, stick_to_shard=False, tensor_init_bytes=1048576):
+
+    Pipeline._current_pipeline._reader = "COCOReader"
```

Comment:
seems like shard_id is passed to fn and not used?

Reply:
Yes, Rajy 
This change was done to match the outer API with arguments same as DALI's .